alma/alma-installments-prestashop
# [insurance] Template cart product details delete subscription

### Reason for change



[Linear task](https://linear.app/almapay/issue/MPP-1120/prestashop-template-cart-product-details-delete-subscription)

### Code changes

Integration design item insurance on cart page

### How to test

Add insurance in your cart and see the integration item
Remove the insurance only with the button remove insurance in the bottom of the item

### Checklist for authors and reviewers




### Non applicable
